lib/models/movie.js

```diff
@@ -4,6 +4,43 @@ const Bookshelf = require('../libraries/bookshelf');
 
 module.exports = Bookshelf.Model.extend({
   tableName: 'movies',
+  filter: function (filter) {
+    if (!filter || !filter.name && !filter.release_year) {
```

Comment:
Can you put parenthesis here? It's hard to figure out both what you meant and what the code is actually doing here.

Reply:
I originally had parenthesis because I also thought it was confusing, but that generated a lint error for excessive parenthesis 😢